src/engines/html5/html5.js

```diff
@@ -907,6 +908,19 @@ export default class Html5 extends FakeEventTarget implements IEngine {
     this.dispatchEvent(new FakeEvent(CustomEventType.TEXT_CUE_CHANGED, {cues: activeCues}));
   }
 
+  /**
+   * set hasBeenReset to true for all the cues. (use case: when cues should be recalculated for display)
+   * @returns {void}
+   */
+  resetAllCues(): void {
+    let activeTextTrack = Array.from(this._el.textTracks).find(track => track && track.mode !== 'disabled');
```

Comment:
what about the tracks that already have cues but are now in disabled mode?

Reply:
they will be reset when the user will select the textTrack which they belong to